### DISCUSSION TITLE
Aligning floating nodes

### DISCUSSION THREAD

**Discussion by JonPolish** [freeplane/freeplane#974] [Help]
I cannot find an option to align selected floating nodes. Is this possible?

Thank you.

Jon

**Comment by euu2021** [comment]
I think this option doesn't exist. But you can increase the value of `Preferences…->Behaviour->Grid gap size`, so it gets easier to align nodes

**Comment by dpolivaev** [comment]
Currently, you can select multiple nodes and move them all simultaneously. 
They are automatically aligned as you move them.

**Reply by euu2021** [reply]
It seems that there is only vertical alignment, and not horizontal alignment?

![javaw_ofB1OHMYXP](https://user-images.githubusercontent.com/77707706/215863119-87a9155a-f4a0-4565-9b2c-63938f6ba14e.gif)

**Reply by JonPolish** [reply]
Floating nodes? Seems I can move only one at a time even after selecting several. As for alignment, it is not working horizontally or vertically. Perhaps I am doing something wrong.

I am running 1.10.6.

Jon

**Reply by dpolivaev** [reply]
Probably it is a new feature on Freeplane 1.11

**Reply by JonPolish** [reply]
I see. Thank you for the information.

Jon
